resources/views/verification.blade.php

```diff
@@ -124,18 +124,16 @@
 <script src="https://cdnjs.cloudflare.com/ajax/libs/popper.js/1.14.3/umd/popper.min.js" integrity="sha384-ZMP7rVo3mIykV+2+9J3UJ46jBk0WLaUAdn689aCwoqbBJiSnjAK/l8WvCWPIPm49" crossorigin="anonymous"></script>
 <script src="https://stackpath.bootstrapcdn.com/bootstrap/4.1.3/js/bootstrap.min.js" integrity="sha384-ChfqqxuZUCnJSK3+MXmPNIyE6ZbWh2IMqE241rYiqJxyMiZ6OW/JmZQ5stwEULTy" crossorigin="anonymous"></script>
 <script type="text/javascript">
-    $(document).ready(function() {
-        $("#resendCodeBtn").click(function() {
-            var xhr = new XMLHttpRequest();
-
-            xhr.open("POST", '{{ \Illuminate\Support\Facades\URL::route('brackets/verifications/resend') }}', true);
-            xhr.setRequestHeader('Content-Type', 'application/json');
-            xhr.setRequestHeader('X-CSRF-TOKEN', $('meta[name="csrf-token"]').attr('content'));
-            xhr.send(JSON.stringify({
-                action_name: $("[name='action_name']").val()
-            }));
-        });
-    });
+    function generateNewCode() {
```

Comment:
Nechceli sme to spraviť bez JS? Len ako <form> ktorý sa submitne po kliku na button.

Reply:
Ano, chceli sme to ma ako form :)

Ale akoze v principe ak je to vanilla JS, tak mi to az tak nevadi, co myslite? Mne vadil hlavne ten boostrap framework.

Ale akoze, ked sme submit nahradili formom, tak aj resendCode by sme mohli nahradit formom, to by nemal byt taky problem.